doc/Manifest.toml

```diff
@@ -72,7 +74,7 @@ deps = ["Distributed", "InteractiveUtils", "Logging", "Random"]
 uuid = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [[UUIDs]]
-deps = ["Random", "SHA"]
+deps = ["Random"]
```

Comment:
What happened here? @KristofferC ?

Reply:
Different Julia versions.

Reply:
In which Julia version does `UUIDs` not depend on `SHA` though?